frontend/packages/ceph-storage-plugin/src/components/bucket-class/modals/_bs-modal.scss

```
@@ -16,6 +16,14 @@
   }
 }
 
+.nb-bc-modal__text{
```

```suggestion
.nb-bc-modal__text{
```

does not seem to modify text ? We can name it something else ?